projects/daq3/common/daq3_qsys.tcl

```diff
@@ -56,15 +79,16 @@ add_connection ad9152_jesd204.link_clk avl_ad9152_fifo.if_dac_clk
 add_connection ad9152_jesd204.link_reset avl_ad9152_fifo.if_dac_rst
 add_connection util_ad9152_upack.if_packed_fifo_rd_en avl_ad9152_fifo.if_dac_valid
 add_connection avl_ad9152_fifo.if_dac_data util_ad9152_upack.if_packed_fifo_rd_data
-add_connection avl_ad9152_fifo.if_dac_dunf axi_ad9152_core.if_dac_dunf
+add_connection avl_ad9152_fifo.if_dac_dunf axi_ad9152_tpl.if_dac_dunf
 
 # ad9152-dma
 
 add_instance axi_ad9152_dma axi_dmac
 set_instance_parameter_value axi_ad9152_dma {DMA_DATA_WIDTH_SRC} {128}
-set_instance_parameter_value axi_ad9152_dma {DMA_DATA_WIDTH_DEST} {128}
+set_instance_parameter_value axi_ad9152_dma {DMA_DATA_WIDTH_DEST} $dac_data_width
 set_instance_parameter_value axi_ad9152_dma {DMA_2D_TRANSFER} {0}
 set_instance_parameter_value axi_ad9152_dma {SYNC_TRANSFER_START} {0}
+set_instance_parameter_value axi_ad9152_dma {CYCLIC} {1}
```

Comment:
We have an offload FIFO in the TX path, setting the DMA to cyclic mode can break the offload functionality.

Reply:
This only enables the DMA capability. It will be useful when bypass will be enabled. Will add bypass capability in the next release